### PR TITLE
Fixes issue #25 - STDOUT/STDERR not being caught

### PR DESCRIPTION
**Pull request by TennisGazelle**
- [x] Does all of your new or changed code have unit tests?
- [ ] Is overall unit test statement coverage still above 85%?
- [x] Is code style linting passing with `npm run lint`?
- [x] Is the related GitHub issue number included in your branch name? ex branch name: `fix/12-some-bug`
- [x] Is the related GitHub issue number included in your pull request title? example PR title: "12 - Fixes some bug"
- [x] Is documentation still up to date after your change?

### CODE REVIEW

File: src/executor.js [hunk 1]
```diff
@@ -94,12 +91,18 @@ export const execute = (command, args = [], opts = {}) => {
...
+      logger.logOutput(childData);
+      processOutput(childData);
     });
 
     child.stderr.on('data', data => {
       const childData = data.toString();
-      logger.logOutput(childData);
-      output += childData;
+      logger.logStderr(childData);
+      processOutput(childData);
```
Comment (ejfrancis) [inline src/executor.js]: :+1:

**Comment by TennisGazelle** [comment]
> @TennisGazelle this seems a little weird. now that there are separate log methods for stdout vs stderr, they should probably be prefixed with "STDERR" and "STDOUT" respectively. Instead of having stdout prefixed with "OUTPUT" and stderr prefixed with "STDOUT" (which could be quite confusing when trying to figure out an issue by browsing logs)

i can fix that.  can i get contributor permissions, even without mergeability rights? (i don't like dealing with forks)

**Comment by worldjoe** [comment]
@TennisGazelle also, add a unit test. Circle CI didn't complain on this PR, but a dependency PR broke due to dropped BRANCH coverage. https://github.com/intuit/judo/pull/24

**Comment by TennisGazelle** [comment]
> @TennisGazelle also, add a unit test. Circle CI didn't complain on this PR, but a dependency PR broke due to dropped BRANCH coverage. #24

ok

**Pull request merged**